api/v1beta1/types.go

```diff
@@ -923,6 +923,17 @@ type VCN struct {
 	NetworkSecurityGroups []*NSG `json:"networkSecurityGroups,omitempty"`
 }
 
+// LoadBalancerType is an enumeration of the supported load balancer types.
+type LoadBalancerType string
+
+const (
+	// NetworkLoadBalancer is the default load balancer type.
+	NetworkLoadBalancer LoadBalancerType = "NLB"
+
+	// ApplicationLoadBalancer is the alternative load balancer type.
+	ApplicationLoadBalancer LoadBalancerType = "LBaaS"
```

Comment:
you dont need the fields in v1beta1

Reply:
Sure, I'll remove this, and also take a look at the conversion function between v1beta1 and v1beta2 to see if any updates are needed.